### PR TITLE
Branch perf v8

### PR DESCRIPTION
**Pull request by aleksbykov**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/remote.py [hunk 1]
```diff
@@ -612,7 +612,7 @@ def _make_scp_cmd(self, src, dst, connect_timeout=300, alive_interval=300):
         key_option = ''
         if self.key_file:
             key_option = '-i %s' % os.path.expanduser(self.key_file)
-        command = ("scp -r -o StrictHostKeyChecking=no -o BatchMode=yes "
+        command = ("scp -vvv -r -o StrictHostKeyChecking=no -o BatchMode=yes "
```
Comment (fgelcer) [inline sdcm/remote.py:615]: the `-vvv` should be removed before merging

File: test-cases/performance/perf-regression.100threads.30M-keys.yaml [hunk 2]
```diff
@@ -24,3 +24,5 @@ send_email: true
 email_recipients: ['qa@scylladb.com']
 
 use_legacy_cluster_init: false
+
+logs_transport: "ssh"
```
Comment (fgelcer) [inline test-cases/performance/perf-regression.100threads.30M-keys.yaml:28]: need an empty line at the end of the file

**Review by fgelcer — COMMENTED** [review]
LGTM, but i hope it works... and that we will be able to backport it to `branch-perf-v12`

**Comment by aleksbykov** [comment]
Staging job with write throughput is passed. Fix according Fabio's comments

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by roydahan — APPROVED** [review]

**Comment by fgelcer** [comment]
@aleksbykov @roydahan , there are some of the tests running on `branch-perf-v10`, and we need to backport either this one (or the one against `branch-perf-v12` to it

**Comment by fgelcer** [comment]
and `branch-perf-v13` too

**Comment by fgelcer** [comment]
and [this job](https://jenkins.scylladb.com/view/master/job/scylla-master/view/master-weekly-performance-tests/job/scylla-master-perf-regression-aws-test-latency/218/) (`scylla-master-perf-regression-aws-test-latency`) failed, probably something else missing or environment error

**Comment by fgelcer** [comment]
same [here](https://jenkins.scylladb.com/view/master/job/scylla-master/view/master-weekly-performance-tests/job/scylla-master-perf-regression-aws-test-throughput-disk-cache/93/)

**Pull request merged**